src/APIBasedAuth.ts

```diff
@@ -227,6 +227,25 @@ class APIBasedAuth {
     return data;
   }
 
+  /**
+   * Confirms and completes a new user's signup. This requires
+   * a confirmation code to prove that the user has access to the
+   * provided email address
+   */
+  public async confirmSignup(
+    input: Omit<APIBasedAuth.ConfirmSignupData, 'clientId'>
+  ) {
+    const { data } = await this.apiClient.post<
+      APIBasedAuth.ConfirmSignupResponse,
+      AxiosResponse<APIBasedAuth.ConfirmSignupResponse>,
+      APIBasedAuth.ConfirmSignupData
+    >('/confirm', {
+      clientId: this.clientOptions.clientId,
+      ...input
+    });
+    return data;
```

Comment:
I think we should call `this.setTokens(...)` before this return, similar to line 202

Reply:
🤦